src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
You can simplify code below by using object destructuring

Reply:
ty for advice